src/frontend/src/components/Header/Header.jsx

```diff
@@ -4,14 +4,19 @@ import useMediaQuery from '@material-ui/core/useMediaQuery';
 
 import MobileHeader from './MobileHeader.jsx';
 import DesktopHeader from './DesktopHeader.jsx';
+import Login from './Login.jsx';
 
 function Header() {
   const theme = useTheme();
   const matches = useMediaQuery(theme.breakpoints.down('sm'));
 
   return (
     <div>
-      {matches ? <MobileHeader className="mobile-header" /> : <DesktopHeader className="header" />}
+      {matches ? (
+        <MobileHeader className="mobile-header" name={Login()} />
+      ) : (
+        <DesktopHeader className="header" name={Login()} />
```

Comment:
What's happening here with `Login()`?  You are trying to send this component down on props?  Why not have your components require this in directly?

Reply:
Made the changes you suggested